bin/check_google_doc_export.rb

```diff
@@ -16,12 +16,22 @@
 require_relative '../lib/article_json'
 
 doc_id = '1E4lncZE2jDkbE34eDyYQmXKA9O26BHUiwguz4S9qyE8'
-url = "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
+url =
+  "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
 expected_document = JSON.parse(File.read('spec/fixtures/reference_document_parsed.json'))
 
 exported_doc = Net::HTTP.get(URI.parse(url))
 document = JSON.parse(ArticleJSON::Article.from_google_doc_html(exported_doc).to_json)
 
+# `source_url` (for hosted images) is dynamic, so we need to remove it from the comparison`
+def nullify_source_url(hash)
+  hash['content'].each { |key, _value| key['source_url'] = nil if key['source_url'] }
```

Comment:
I think `key` isn't the right name to use here. Assuming that `hash['content']` is an array of hashes (which are the objects we call elements), this might be a clearer alternative:
```suggestion
  hash['content'].each { |element| element['source_url'] = nil if element['source_url'] }
```

Reply:
This is cleaner and I have amended it to this. Thanks.